src/ops/mod.rs

```diff
@@ -49,16 +57,16 @@ impl ExitError {
     /// Helpers to create exit results
     ///
     /// Note: err panics if exitcode is zero.
-    fn err<T>(exitcode: i32, message: T) -> OpResult
+    fn err<T>(exitcode: i32, message: T) -> ExitError
```

Comment:
(note for future graham: why are we changing OpResult to ExitError?)

Reply:
Returning an `OpResult` didn’t work in all cases I need `err` for, because I needed to construct an `ExitError` in some cases.